packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -6,13 +6,14 @@
 /// @docImport 'sliver_list.dart';
 library;
 
+import 'dart:collection';
 import 'dart:math' as math;
 
+import 'package:collection/collection.dart';
 import 'package:flutter/foundation.dart';
-
-import 'box.dart';
-import 'sliver.dart';
-import 'sliver_multi_box_adaptor.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/src/widgets/indexed_scroll_controller.dart';
+import 'package:flutter/src/widgets/scroll_position.dart';
```

Comment:
This is not in line with Flutter's layered architecture. The rendering library can't import files from the widgets library.